index.html

```diff
@@ -17,15 +18,20 @@
     <!-- header and headline -->
 
     <section class="first-page">
-        <header>
+        <!-- <header> -->
             <nav id="head">
-                <a href="#">Safar Ali</a>
+                <a class="logo" href="#">Safar Ali</a>
                 <a href="#"><img src="./images/Union.png" alt="burger-icon"></a>
+                <ul class="menu-link">
+                    <li><a href="#">Projects</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#"> Contact </a></li>
+                </ul>
             </nav>
-        </header>
+        <!-- </header> -->
```

Comment:
- [ ] [OPTIONAL] Please, consider removing the comments to make your code cleaner 👍